internal/config/mount_config.go

```diff
@@ -65,10 +66,11 @@ type MetadataCacheConfig struct {
 	// no cache and > 0 for ttl-controlled metadata-cache.
 	// Any value set below -1 will throw an error.
 	TtlInSeconds int64 `yaml:"ttl-secs,omitempty"`
-	// TypeCacheMaxEntries is the upper limit
-	// on the maximum number of type-cache entries,
-	// which are maintained at per-directory level.
-	TypeCacheMaxEntries int `yaml:"type-cache-max-entries,omitempty"`
+	// TypeCacheMaxSizeMb is the upper limit
+	// on the maximum size of type-cache maps,
+	// which are currently
+	// maintained at per-directory level.
+	TypeCacheMaxSizeMb int `yaml:"type-cache-max-size-mb,omitempty"`
```

Comment:
would prefer using "B" instead of "b" in "Mb". it will also become consistent to MaxSizeInMB in FileCache

Reply:
done